azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/CallCompositeActivity.kt

```diff
@@ -69,13 +70,25 @@ internal class CallCompositeActivity : AppCompatActivity() {
 
         window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
         configureActionBar()
+        configureLocalization()
         setStatusBarColor()
         setActionBarVisibility()
+
         if (configuration.themeConfig?.theme != null) {
             theme.applyStyle(
                 configuration.themeConfig?.theme!!, true
             )
         }
+
+        diContainerHolder.localizationConfiguration = configuration.localizationConfig
```

Comment:
can this be a getter inside diContainerHolder instead of assigned inside the Activity?